server/channels/store/sqlstore/channel_store.go

```diff
@@ -2069,6 +2059,32 @@ func (s SqlChannelStore) GetMember(ctx context.Context, channelID string, userID
 	return dbMember.ToModel(), nil
 }
 
+func (s SqlChannelStore) GetMemberOnly(ctx context.Context, channelID string, userID string) (*model.ChannelMember, error) {
+	var dbMember model.ChannelMember
+	if err := s.DBXFromContext(ctx).Get(&dbMember, `SELECT ChannelId,
+		UserId,
+		Roles,
+		LastViewedAt,
+		MsgCount,
+		MentionCount,
+		MentionCountRoot,
+		COALESCE(UrgentMentionCount, 0) AS UrgentMentionCount,
+		MsgCountRoot,
+		NotifyProps,
+		LastUpdateAt,
+		SchemeUser,
+		SchemeAdmin,
+		SchemeGuest FROM ChannelMembers
+		WHERE ChannelId=? AND UserId=?`, channelID, userID); err != nil {
```

Comment:
Nit: can we move the `FROM` part to a new line?
```suggestion
	if err := s.DBXFromContext(ctx).Get(&dbMember, `SELECT ChannelId,
		UserId,
		Roles,
		LastViewedAt,
		MsgCount,
		MentionCount,
		MentionCountRoot,
		COALESCE(UrgentMentionCount, 0) AS UrgentMentionCount,
		MsgCountRoot,
		NotifyProps,
		LastUpdateAt,
		SchemeUser,
		SchemeAdmin,
		SchemeGuest
		FROM ChannelMembers
		WHERE ChannelId=? AND UserId=?`, channelID, userID); err != nil {
```